src/config/dynamic-theme-fixes.config

```diff
@@ -18714,6 +18714,22 @@ IGNORE INLINE STYLE
 
 ================================
 
+terraria.wiki.gg
+
+CSS
+:root {
+    --darkreader-bg--theme-content-background: transparent;
+    --darkreader-bg--ooui--menuselect-background-color: #111111;
```

Comment:
IMO the generated colors seems to be correct, given the non-modified site's colors.

Reply:
`--darkreader-bg--theme-content-background: transparent;`
This is to get rid of a lot of the mid-brown background colors. Here's an alternative that has the same functionality.

```css
.mw-body, .infocard, .infocard .main-heading .hgroup {
    background: none;
}
```

`--darkreader-bg--ooui--menuselect-background-color`
This change is for the popup in when typing in the search bar to make it match the rest of the site background.